### PR TITLE
Invoke webhooks on game start and phase start

### PR DESCRIPTION
**Pull request by johnpooch**

### CODE REVIEW

File: game/game.go [hunk 1]
```diff
@@ -1142,6 +1143,27 @@ func asyncStartGame(ctx context.Context, gameID *datastore.Key, host string) err
 		}
 		g.ID = gameID
 
+		discordBotToken, err := auth.GetDiscordBotToken(ctx)
```
Comment (zond) [inline game/game.go:1146]: The stuff happening here and in phase.go are _very_ similar. Perhaps extract them to some "call discord webhook with given data"-function?

Comment (johnpooch) [reply game/game.go]: Updated. Let me know if you're happy with this approach and I'll merge. Have tested against local container and it works

**Comment by zond** [comment]
I think it looks good.

File: game/game.go (hunk 1 above)
Comment (zond) [reply game/game.go]: :+1:

**Pull request merged**